cmd/minishift/cmd/start.go

```diff
@@ -568,6 +571,7 @@ func initStartFlags() *flag.FlagSet {
 	startFlagSet.String(configCmd.Memory.Name, constants.DefaultMemory, "Amount of RAM to allocate to the Minishift VM. Use the format <size><unit>, where unit = MB or GB.")
 	startFlagSet.String(configCmd.DiskSize.Name, constants.DefaultDiskSize, "Disk size to allocate to the Minishift VM. Use the format <size><unit>, where unit = MB or GB.")
 	startFlagSet.String(configCmd.HostOnlyCIDR.Name, "192.168.99.1/24", "The CIDR to be used for the minishift VM. (Only supported with VirtualBox driver.)")
+	startFlagSet.Bool(configCmd.SkipCheckPreflight.Name, false, "Skip the Preflight checks")
```

Comment:
Please put `.` i.e. full stop at the end of the sentence. Check the other usage messages for reference
```
--server-loglevel int              Log level for the OpenShift server.
      --skip-preflight-checks            Skip the Preflight checks
      --skip-registration                Skip the virtual machine registration.
      --skip-registry-check              Skip the Docker daemon registry check.
```